js_modules/dagit/packages/core/src/assets/AssetActionMenu.tsx

```diff
@@ -17,56 +18,69 @@ interface Props {
 
 export const AssetActionMenu: React.FC<Props> = (props) => {
   const {repoAddress, asset, onWipe} = props;
-  const {canWipeAssets} = usePermissions();
+  const {canWipeAssets, canLaunchPipelineExecution} = usePermissions();
   const {path} = asset.key;
 
+  const {onClick, loading, launchpadElement} = useMaterializationAction([asset.key]);
```

Comment:
@bengotow Should this have a "preferred job name" value?  I wasn't sure where I might get that from, or whether I need it here.

Reply:
Hey! Talked about this offline but I think it's fine to omit this in this context and have the button launch runs of the hidden asset jobs and not a specific user-managed asset job.